### PR TITLE
Important bugfix: Read dz from input

### PR DESCRIPTION
**Pull request by bendudson**
In the move to v4, the code which read "dz" was clobbered.
This broke blob2d example, and probably some others.

### CODE REVIEW

File: examples/blob2d/blob2d.cxx [hunk 1]
```diff
@@ -128,7 +128,7 @@ class Blob2D : public PhysicsModel {
     // Density Evolution
     /////////////////////////////////////////////////////////////////////////////
 
-    ddt(n) = -bracket(phi,n,BRACKET_ARAKAWA)    // ExB term
+    ddt(n) = -bracket(phi,n,BRACKET_SIMPLE)    // ExB term
```
Comment (loeiten) [inline examples/blob2d/blob2d.cxx]: Why the change to `BRACKET_SIMPLE`? Would this not cause a blow up of terms?

File: examples/blob2d/blob2d.cxx [hunk 2]
```diff
@@ -143,7 +143,7 @@ class Blob2D : public PhysicsModel {
     // Vorticity evolution
     /////////////////////////////////////////////////////////////////////////////
 
-    ddt(omega) = -bracket(phi,omega, BRACKET_ARAKAWA)                     // ExB term
+    ddt(omega) = -bracket(phi,omega, BRACKET_SIMPLE)                     // ExB term
```
Comment (loeiten) [inline examples/blob2d/blob2d.cxx]: The same here.

File: examples/blob2d/blob2d.cxx (hunk 1 above)
Comment (bendudson) [inline examples/blob2d/blob2d.cxx]: Reverted to SIMPLE because that was what was used in v3.1. The ARAKAWA bracket has less dissipation, so results in more noisy solutions which seems less robust than SIMPLE. Since this is one of the easier examples to start with, I thought it better to have it as robust as possible.
